src/main/fc/runtime_config.h

```diff
@@ -109,6 +109,19 @@ extern uint8_t stateFlags;
 uint16_t enableFlightMode(flightModeFlags_e mask);
 uint16_t disableFlightMode(flightModeFlags_e mask);
 
+typedef struct {
+    uint32_t timestamp;
```

Comment:
What units is this? Presumably microseconds, if so define it as: `timeUs_t timestampUs`.